tests/expectations/tests/class_nested.rs

```diff
@@ -73,6 +53,26 @@ fn bindgen_test_layout_A() {
 impl Clone for A {
     fn clone(&self) -> Self { *self }
 }
+#[repr(C)]
+#[derive(Debug, Default, Copy, Hash, PartialEq, Eq)]
+pub struct C {
```

Comment:
This should still be `A_C` though, shouldn't it? I don't think this change is desirable.

```c++
class A {
public:
    int member_a;
    class B {
        int member_b;
    };

    class C;

    template<typename T>
    class D {
      T foo;
    };
};

class A::C {
  int baz;
};
```